scripts/main-yield-charts.main.v1.0.0.js

```diff
@@ -0,0 +1,257 @@
+const MOTNHS = ['Ene', 'Feb', 'Mar', 'Abr', 'May', 'Jun', 'Jul', 'Ago','Sept', 'Oct', 'Nov', 'Dic']
+var options = {
+  width: '100%',
+  height: 250,
+  colors: ['#25D16A'],
+  maintainAspectRatio: false,
+  legend: {
+    position:'none'
+  },
+  vAxis: {
+    format: "#.##'%'",
+    textStyle:{color: '#74797f'}
+  }, 
+  hAxis: {
+    textStyle:{color: '#74797f'}
+  },
+  pointSize: 5,
+  chartArea: {
+    height: '100%',
+    width: '100%',
+    top: 48,
+    left: 48,
+    right: 16,
+    bottom: 48
+  },
+  height: '100%',
+  width: '100%',
+};
+const numberToPercent = (number) => ((number - 1) * 100).toFixed(2) + '%';
```

Comment:
Ahora que podemos poner espacios, lo haría jeje
```suggestion
};

const numberToPercent = (number) => ((number - 1) * 100).toFixed(2) + '%';
```